src/main/resources/schema/ans/0.10.5/traits/trait_dynamic_items.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.5/traits/trait_dynamic_items.json",
+  "title": "Dynamic Items",
+  "description": "Dynamic items for Collection Query Backfill",
+  "type": "object",
+  "properties": {
+    "type": {
+      "title": "Type",
+      "description": "Section or Tag for query backfilling",
+      "type": "string",
+      "enum": ["sections", "tags"]
+    },
+    "ids": {
+      "description": "A list of Section or Tag IDs that will be used to Backfill queries.",
+      "type": "array",
+      "maxItems": 100,
```

Comment:
Assuming my emended comment on the proposal is accurate, this is probably OK: https://github.com/washingtonpost/ans-schema/pull/231#discussion_r472453352

Reply:
yes, that's the right understanding